[Automated] Add w4po.ExplorerTabUtility version 2.3.0

w4po opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/243719)

w4po committed:
[Automated] Add w4po.ExplorerTabUtility version 2.3.0

stephengillie commented:
Automatic Validation ended with:
>  Signature Update Failed. Failure to parse win 10, attempting win 11 Installation failed with exit code -1978335216 2025-03-30 18:24:02.826 [CORE] Deployment operation #0: error 0x800700EA: Failed to load the extension DLL due to the following error: More data is available. 2025-03-30 18:24:02.826 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!5FA4DF27: (caller: 5FA4E606) Exception(1) tid(14dc) 800700EA More data is available.     Msg:[Operation failed: error 0x800700EA: Failed to load the extension DLL due to the following error: More data is available.     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!5FA4DF27: (caller: 5FA4E606) Exception(1) tid(14dc) 800700EA More data is available. 2025-03-30 18:24:02.846 [REPO] Source add/update failed, waiting 5518 milliseconds and retrying: winget
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 0000000B | 11 | -4294967285 | ERROR_BAD_FORMAT | An attempt was made to load a program with an incorrect format. | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 


(Automated response - build 1031.)

w4po commented:
​Hello @stephengillie,
I hope you're doing well.
As with previous submissions, the automatic validation for PR [#243719](https://github.com/microsoft/winget-pkgs/pull/243719) has flagged an issue due to my application's Windows 11 requirement. Could you please assist with a manual review when you have a moment?
Your help is greatly appreciated as always ♥️

stephengillie commented:
Manual Validation ended with:
> 9:50:48 PM:  finished with exit code: -1978335216      
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 

 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 


(Automated response - build 1034.)

ItzLevvie commented:
@wingetbot run

w4po commented:
It needs windows 11.

ItzLevvie commented:
@wingetbot run

w4po commented:
> @wingetbot run

Hi,
Thank you for your contribution and your time,

My app has a history of failing the automatic validation, it won't work, I don't know why, maybe because it needs Win11...

[PR 221627](https://github.com/microsoft/winget-pkgs/pull/221627) [PR 228779](https://github.com/microsoft/winget-pkgs/pull/228779) [PR 233771](https://github.com/microsoft/winget-pkgs/pull/233771)

But it works manually.

w4po commented:
>>><<<<

> > @wingetbot run
> 
> Hi, Thank you for your contribution and your time,
> 
> My app has a history of failing the automatic validation, it won't work, I don't know why, maybe because it needs Win11...
> 
> [PR 221627](https://github.com/microsoft/winget-pkgs/pull/221627) [PR 228779](https://github.com/microsoft/winget-pkgs/pull/228779) [PR 233771](https://github.com/microsoft/winget-pkgs/pull/233771)
> 
> But it works manually.

Utesgui commented:
Can confirm the installtion does nothing on Windows Server 2022 but installs fine on Windows Server 2025:
![image](https://github.com/user-attachments/assets/a21e6d0c-71af-4cdc-a62b-28fb09ea2b93)

stephengillie commented:
Automatic Validation ended with:
>  Signature Update Failed. Failure to parse win 10, attempting win 11 Installation failed with exit code -1978335216 2025-04-05 20:30:02.397 [CORE] Deployment operation #0: error 0x800700EA: Failed to load the extension DLL due to the following error: More data is available. 2025-04-05 20:30:02.398 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!60F4DF27: (caller: 60F4E606) Exception(1) tid(3dc) 800700EA More data is available.     Msg:[Operation failed: error 0x800700EA: Failed to load the extension DLL due to the following error: More data is available.     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!60F4DF27: (caller: 60F4E606) Exception(1) tid(3dc) 800700EA More data is available. 2025-04-05 20:30:02.421 [REPO] Source add/update failed, waiting 2109 milliseconds and retrying: winget
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 0000000B | 11 | -4294967285 | ERROR_BAD_FORMAT | An attempt was made to load a program with an incorrect format. | 
 | 0000083D | 2109 | -4294965187 | ERROR_CONNECTED_OTHER_PASSWORD_DEFAULT | The network connection was made successfully using default credentials. | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 


(Automated response - build 1042.)

w4po commented:
@stephengillie

stephengillie commented:
Automatic Validation ended with:
>  Signature Update Failed. Failure to parse win 10, attempting win 11 Installation failed with exit code -1978335216 2025-04-16 15:16:19.976 [CORE] Deployment operation #0: error 0x800700EA: Failed to load the extension DLL due to the following error: More data is available. 2025-04-16 15:16:19.976 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!621FDF27: (caller: 621FE606) Exception(1) tid(d4c) 800700EA More data is available.     Msg:[Operation failed: error 0x800700EA: Failed to load the extension DLL due to the following error: More data is available.     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!621FDF27: (caller: 621FE606) Exception(1) tid(d4c) 800700EA More data is available. 2025-04-16 15:16:19.998 [REPO] Source add/update failed, waiting 3241 milliseconds and retrying: winget
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 0000000B | 11 | -4294967285 | ERROR_BAD_FORMAT | An attempt was made to load a program with an incorrect format. | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 


(Automated response - build 1045.)

stephengillie commented:
Manual Validation ended with:
> 
> 10:29:07 PM:  finished with exit code: -1978335216                                                                      10:29:18 PM: Install Failed.               
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 

 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 


(Automated response - build 1045.)

w4po committed:
Merge branch 'microsoft:master' into w4po.ExplorerTabUtility-2.3.0-a91aa219-3338-4a59-abe6-d0ce4384e235

w4po committed:
refactor: remove MinimumOSVersion and add requirement in description (winget)

- Remove MinimumOSVersion: 10.0.22621.0 from installer manifest to avoid
  manual review delays in winget-pkgs validation
- Add Windows 11 requirement information to ShortDescription in locale manifest
- Ensures users are informed about OS requirements while streamlining the
  package acceptance process

w4po commented:
refactor: 
- Remove MinimumOSVersion: 10.0.22621.0 from installer manifest to avoid
  manual review delays in winget-pkgs validation
- Add Windows 11 requirement information to ShortDescription in locale manifest
